app/views/dish/edit.html.haml

```diff
@@ -0,0 +1,19 @@
+.row
+  .medium-8.medium-centered
+    - if @dish.errors.any?
+      .callout.small.alert
+        %ul
+          Please make sure to fill in all the fields correctly
+        - @dish.errors.full_messages.each do |error|
+          %li
+            = error
+    = form_for [ @menu.restaurant, @menu, @dish ], url: restaurant_menu_dish_path do |f|
```

Comment:
This is repetitive considering we already have a partial containing all this fields.